acme/authority.go

```diff
@@ -254,32 +273,187 @@ func (a *Authority) GetAuthz(p provisioner.Interface, accID, authzID string) (*A
 	return az.toACME(a.db, a.dir, p)
 }
 
-// ValidateChallenge attempts to validate the challenge.
```

Comment:
The documentation should start with `ValidateChallenge ...`

Reply:
Yup ran into this elsewhere. While I don't understand this lint rule, it's not hard to conform. Do you have any background on why this lint rule exists/is important?

Reply:
quoting mariano: 

This rule is also not as a custom configuration that we set up, it's the default and it is used all over the standard library and 3rd party libraries in Go, see for example https://golang.org/pkg/crypto/x509/#CreateCertificate

Reply:
Instead of just `ValidateChallenge ...` to pass the linter you can use the previous command, and then write the full docs that you've written.

Reply:
I mean the reason I changed it in the first place is that restating name of the function in a sentence does not documentation make. `ValidateChallenge attempts to validate the challenge.` is entirely redundant and superfluous and tells the reader nothing. So I wrote a nice explanation of what the func is doing etc. I'll write a more descriptive intro sentence that begins with ValidateChallenge.